lib/Conversation/CommentList.js

```diff
@@ -5,13 +5,13 @@ const CommentList = ({ comments, ...rest }) => {
   const list = comments.map(comment =>
```

Comment:
`comments` is a required prop in the parent component so I don't think it needs a default prop. Instead I'll make it required 👍